### DISCUSSION TITLE
is there a roadmap for 2024 or next few years?

### DISCUSSION THREAD

**Discussion by hudiehule** [kubevela/kubevela#6620] [General]
I would like to know about kubevela’s development plan in the next few years. is there a roadmap for 2024 or next few years?

**Comment by tedchang77** [comment]
the lack of response to issues and questions is concerning. i'm not sure kubevela is something teams should start using unless the owners make it clear that this project will be continue to be supported.

**Comment by jguionnet** [comment]
See https://kubevela.io/docs/roadmap/ for 2026 roadmap and onwards. Join community meeting and slack for questions and discussion

Here are a few of the KEPs we've recently published and we would love feedback on

https://github.com/kubevela/kubevela/pull/6996 (Declarative Addons / Addons-as-Components)
https://github.com/kubevela/kubevela/pull/6993 (Component Layering / Compositions - with other definition types to follow!)
https://github.com/kubevela/kubevela/pull/6974 (Native Helm Component - with health eval!)
